### ISSUE TITLE
Audinaut doesn't connect to local https server (navidrome behind nginx)

### ISSUE THREAD

**Issue by IeP4nieF** [nvllsvm/Audinaut#100]
I'm trying to configure audinaut to use my local navidrome-server. The server is behind a reverse proxy (nginx), which redirects also every traffic to port 80 to port 443. The nginx-reverse-proxy uses a local certificate, which was made by a local CA. The root-certificate is imported in the android system.

I configured the Server adress in the audinaut settings (https://local-server-name.local-domain). But when I use "Test connection", I get "Connection failure. A network error occured. Please check the server adress or try again later."

From the nginx-logs I can see, that no connection is made by audinaut to the server.

Changing the url to http://... instead of https:// I can see the connection. Then audinaut fails again, because it is redirected to https://....

logcat: https://pastebin.com/PS1ffXev

I think this is a bug, because audinaut doesn't open a connection to the server.

**Comment by Extarys** [comment]
I use a Let's encrypt SSL certificate with a domain. I put my domain under "Server address". I also configured my pihole so when I'm at home it uses the internal IP and not the internet IP.

Have you tried putting the IP address directly? Also maybe try in Local network address in audinaut and select your SSID.

**Comment by IeP4nieF** [comment]
Yes, I tried both. Entering the IP directly (in Server address" and "Local network adress") and entering the FQDN in both fields. Audinout doesn't open a https-connection to navidrome, as I can see from the access.log of the reverse proxy.

**Comment by helfio** [comment]
I'm having a very similar issue with navidrome and audinaut: I have a domain with a let's encrypt cert. This domain has a VPS behind with a reverse proxy forwarding traffic to a server I have at home. While I'm at home, I use the local IP thanks to pi-hole DNS capabilities and still I'm able to reuse let's encrypt certificate (i.e. I can check it with a web browser, everything is running just fine). Still, Audinaut cannot connect to the server from the devices I can do it using a web browser.

Any update in your use case, @IeP4nieF ?

**Comment by IeP4nieF** [comment]
No, no updates in my use case. But that isn't surprising, because there wasn't any update of audinaut in f-droid since my bug-report.
